lib/media/ver10/profile/analytics_engine_configuration.ex

```diff
@@ -7,6 +7,7 @@ defmodule Onvif.Media.Ver10.Profile.AnalyticsEngineConfiguration do
   import Ecto.Changeset
   import SweetXml
 
+  require Logger
```

Comment:
unused `require`

Reply:
removed it here https://github.com/hammeraj/onvif/pull/67 @yuriploc